pkgs/development/node-packages/node-packages.nix

```diff
@@ -34375,7 +34375,7 @@ let
       version = "2.5.2";
       src = fetchurl {
         url = "https://registry.npmjs.org/jsesc/-/jsesc-2.5.2.tgz";
-        sha512 = "OYu7XEzjkCQ3C5Ps3QIZsQfNpqoJyZZA99wd9aWd05NCtC5pWOkShK2mkL6HXQR6/Cy2lbNdPlZBpuQHXE63gA==";
+        sha512 = /OYu7XEzjkCQ3C5Ps3QIZsQfNpqoJyZZA99wd9aWd05NCtC5pWOkShK2mkL6HXQR6/Cy2lbNdPlZBpuQHXE63gA==";
```

Comment:
I don't think this is valid...

Reply:
yea i noticed, just a silly little mistake : D

Reply:
that's why i want to run the autogenerate instead, my sleepy head is not great to make manual changes

Reply:
I can't believe i made a mistake twice, I'm so sorry...